src/models/ledger/signer_list.ts

```diff
@@ -0,0 +1,17 @@
+interface SignerEntry {
+  SignerEntry: {
+    Account: string
+    SignerWeight: number
+  }
+}
```

Comment:
Is this supposed to have a nested 'SignerEntry:'? (I didn't see that in the other interfaces you used for sub-interfaces, but I might be missing something)

Link to documentation for reference: https://xrpl.org/signerlist.html#signer-entry-object

Reply:
It's kind of weird, but this one does (this setup is in a couple of other places as well). You can see it in the examples.